applepay/utils.py

```diff
@@ -54,6 +76,24 @@ def signing_time_is_valid(signature, current_time, threshold):
     unexpected format, inconsistent with the CMS 'ContentInfo' object.
     """
     signing_time = retrieve_signature_signing_time(signature)
+    return valid_signing_time(signing_time, current_time, threshold)
+
+
+def valid_signing_time(signing_time, current_time, threshold):
```

Comment:
I don't think having this be a private method adds anything.  It has a clear docstring and single purpose.  If somebody using this lib has a reason to call it directly I don't feel the lib itself needs to be opinionated about that.